spec.md

```diff
@@ -360,6 +351,7 @@ both appear within the same JSON object.
 ### schemaURL
 * Type: `URI`
 * Description: A link to the schema that the `data` attribute adheres to.
+Incompatible changes to the schema SHOULD result in a new schema version.
```

Comment:
I wonder if we should say "URI" instead of "version" just to keep it more generic and not imply that we're mandating a version string be part of the URI, even though that would be a good idea.  What do people think? I'm ok either way- it just jumped out at me.